skyportal/handlers/api/followup_request.py

```diff
@@ -596,3 +622,406 @@ def delete(self, request_id):
             payload={"obj_key": internal_key},
         )
         return self.success()
+
+
+def observation_schedule(
+    followup_requests,
+    instrument,
+    observation_start=Time.now(),
+    observation_end=Time.now() + TimeDelta(12 * u.hour),
+    output_format='csv',
+    figsize=(10, 8),
+):
+
+    """Create a movie to display observations of a given skymap
```

Comment:
Movie?

Reply:
If we were a movie.... you'd be the right guy!